apps/ui/src/components/Ui/InputString.vue

```diff
@@ -29,6 +29,25 @@ const inputValue = computed({
   }
 });
 
+// From AJV string length validation
+// See https://github.com/ajv-validator/ajv/blob/master/lib/vocabularies/validation/limitLength.ts
```

Comment:
Better link would be https://github.com/ajv-validator/ajv/blob/master/lib/runtime/ucs2length.ts I think.